modules/compiler/targets/host/source/target.cpp

```diff
@@ -73,9 +73,13 @@ static llvm::TargetMachine *createTargetMachine(llvm::StringRef CPU,
     Options.MCOptions.ABIName = ABI;
   }
 
+  // TODO: Aarch64 fails on UnitCL test
+  // Execution/Execution.Barrier_02_Barrier_No_Duplicates/OfflineOpenCLC if we
+  // don't set `JIT` to true.
   return LLVMTarget->createTargetMachine(
-      Triple, CPU, Features, Options, llvm::Reloc::Model::Static,
-      llvm::CodeModel::Small, multi_llvm::CodeGenOptLevel::Aggressive);
+      Triple, CPU, Features, Options, /*RM */ std::nullopt,
+      /*CM*/ std::nullopt, multi_llvm::CodeGenOptLevel::Aggressive,
+      /*JIT*/ true);
```

Comment:
Setting `JIT=true` for the non-JIT case seems wrong. Can we take a deeper look at what's going on? (That aside, nit: LLVM style is `/*RM=*/std::nullopt` rather than `/* RM */ std::nullopt`, according to an example in https://llvm.org/docs/CodingStandards.html#comment-formatting.)

Reply:
I agree, I was just looking for a build fix while we investigate.